react/week2/todo-list-app/src/Components/AddTodo.js

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
useState defined, but never used